tox.ini

```diff
@@ -110,15 +109,15 @@ deps =
 commands =
     # workaround as ipopt has incomplete build
     pytest --cov=pypesto --cov-report=xml --cov-append \
-        test/hierarchical -s
+        test/hierarchical
 description =
     Test hierarchical optimization module
 
 [testenv:select]
 extras = test,amici,petab,select
 commands =
     pytest --cov=pypesto --cov-report=xml --cov-append \
-        test/select -s
```

Comment:
not sure but don't we become "less silent" by setting the capture method from `no` to `fd`?

Reply:
Compare:
* now: https://github.com/ICB-DCM/pyPESTO/actions/runs/6912941885/job/18809255758
* before: https://github.com/ICB-DCM/pyPESTO/actions/runs/6905429814/job/18788171857